### ISSUE TITLE
Unable to run Replit starter project 

### ISSUE THREAD

**Issue by dglazkov** [breadboard-ai/breadboard#836]
1. Fork the Replit [Breadboard Starter Project](https://replit.com/@dglazkov/Breadboard-Starter-Project)
2. Hit Run
3. Get this error:

```
> @google-labs/breadboard-hello-world@1.2.5 dev
> breadboard debug src/ --watch -n

node:internal/modules/esm/resolve:844
  throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base), null);
        ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'commander' imported from /home/runner/test/node_modules/@google-labs/breadboard-cli/dist/src/index.js
    at packageResolve (node:internal/modules/esm/resolve:844:9)
    at moduleResolve (node:internal/modules/esm/resolve:901:20)
    at defaultResolve (node:internal/modules/esm/resolve:1121:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:396:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:365:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
    at link (node:internal/modules/esm/module_job:84:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v20.10.0
```

**Comment by dglazkov** [comment]
I think it might be because of this:

```json
  "bundleDependencies": [
    "@google-labs/breadboard",
    "@google-labs/breadboard-web",
    "@google-labs/core-kit",
    "@google-labs/template-kit",
    "commander",
    "esbuild",
    "serve",
    "vite",
    "yaml"
  ]
```

**Comment by dglazkov** [comment]
The "commander", "esbuild", "serve", "vite", or "yaml" aren't bundled in the build process. They are proper deps.

**Comment by Mearman** [comment]
I've fixed it over in my branch for the CLI https://github.com/breadboard-ai/breadboard/pull/793
I want to explore further as this 100% was occurring before we added the `bundleDependencies` to try and resolve it.

**Comment by dglazkov** [comment]
> I've fixed it over in my branch for the CLI #793 I want to explore further as this 100% was occurring before we added the `bundleDependencies` to try and resolve it.

My intuition is that the `@google-labs/*` ones are indeed bundled dependencies. But the rest aren't. No need to remove the whole thing, just the ones that aren't bundled.

**Comment by Mearman** [comment]
@dglazkov I shall update the branch for https://github.com/breadboard-ai/breadboard/pull/793 and test

**Comment by dglazkov** [comment]
Fixed in #793.
